src/static/js/pad_editbar.js

```diff
@@ -392,6 +392,38 @@ var padeditbar = (function()
 
     toolbar.registerCommand("import_export", function () {
       toolbar.toggleDropDown("import_export", function(){
+
+        // does an author exist for this browser??
+        var hasEdits = false;
+
+        // we have a global variable for if the user has edited this pad
+        if (clientVars.thisUserHasEditedThisPad) {
+          hasEdits = true;
+        } else {
+          // get historical edits (not useful for current edits);
+          if (clientVars.collab_client_vars.historicalAuthorData) {
+            $.each(clientVars.collab_client_vars.historicalAuthorData, function(authorId) {
+              if (authorId === clientVars.userId) {
+                clientVars.thisUserHasEditedThisPad = true;
+
+                /*
+                 * I'm setting this because it might be useful in the future.
+                 * Also, we don't want to iterate authors every time we click
+                 * import/export
+                 */
+                hasEdits = true;
+              }
+            });
+          }
```

Comment:
This code covers the case in which we open a pad (or hit F5) where we have already contributed.

Shoudn't be the server to already send us the correct value for `thisUserHasEditedThisPad`? He is the one setting `historicalAuthorData`, after all.

Reply:
I don't see the gain.  Either the server sends a boolean or we iterate the object and set the boolean.  Either way the boolean is set.